bin/spack-install-helper-40-concretize-install

```diff
@@ -15,7 +15,23 @@ do
     spack repo add "${repo}"
 done
 
-spack -e . gpg trust /tmp/spack_sign_key.gpg || echo 'Warning: Could not add spack sign key, i.e. your packages cannot be added to the buildcache'
+# TODO: Remove the Spack version check when we decide to dicsontinue support for older versions
+SPACK_VER=`spack --version`
+IFS="." read SPACK_MAJOR SPACK_MINOR SPACK_PATCH <<< $SPACK_VER
+
+# GPG key is available (we are running in the CI pipeline) => we can push into the buildcache
+if [ -f /tmp/spack_sign_key.gpg ];
```

Comment:
This is maybe not enough to test - Any container that was built with CI-Ext will have this file, but it will be an empty file. I think `[ -s /tmp/spack_sign_key.gpg ]` would be safer as test.